spec/web-server.spec.js

```diff
@@ -0,0 +1,259 @@
+describe("web-server", function () {
+    "use strict";
+
+    var fs, logger, restify, restMiddleware, webServer, WebServer;
+
+    WebServer = require("../lib/web-server");
+
+    beforeEach(function () {
+        fs = {
+            readFileSync: function () {}
+        };
+
+        logger = {
+            console: function () {},
+            debug: function () {},
+            error: function () {},
+            info: function () {},
+            warn: function () {}
+        };
+
+        restify = {
+            createServer: function () {
+                return {};
+            }
+        };
+
+        restMiddleware = function () {
+
+        };
+
+        spyOn(fs, "readFileSync");
+        spyOn(logger, "console");
+        spyOn(logger, "debug");
+        spyOn(logger, "error");
+        spyOn(logger, "info");
+        spyOn(logger, "warn");
+        spyOn(restify, "createServer").andReturn({});
+    });
+
+    beforeEach(function () {
```

Comment:
Can this get merged with the above?  If so, can you drop some of those extra variables so they are scoped to only the callback inside the beforeEach?